More robust CUISequenceSimpleItem loading to prevent crashes when loading game tutorials

sobkas opened the pull request:
Added NULL-checks, skipping not working subitems, increased verbosity of messages
Starting New Game Clear Sky with this change works
Need to also test COP

sobkas committed:
[ClearSky Bugfix]: Fix crash while starting a new game

Added NULL-checks, skipping not working subitems, increased verbosity of messages

Xottab-DUTY commented on src/xrGame/ui/UIGameTutorial.cpp:
Changes to this file totally not needed.
If there's no `game_tutorials.xml`, an assert will be triggered in `CUIXml::Load`. Other changes that add additional logging actually pollute the log.

Xottab-DUTY commented on src/xrGame/ui/UIGameTutorialSimpleItem.cpp:
```diff
@@ -42,8 +42,12 @@ CUIWindow* find_child_window(CUIWindow* parent, const shared_str& _name)
+        {
             return (*_I);
+        }
+    }
```
This is not needed too

sobkas replied:
Fixed

Xottab-DUTY commented on src/xrGame/ui/UIGameTutorial.cpp:
```diff
@@ -114,6 +114,10 @@ bool CUISequencer::Start(LPCSTR tutor_name)
+    else
+    {
+        Msg("found tutorial [%s]", tutor_name);
+    }
```
This is not needed too.
```suggestion
```

sobkas replied:
Fixed

Xottab-DUTY committed:
Reworked subitem loading

Don't add missing subitems to the array at all.

Removed nullptr checks in SSubItem::Start() and SSubItem::Stop(), because we assume that, when we enter these functions, m_wnd never should be nullptr.
